src/codegen.cpp

```diff
@@ -5106,20 +5112,15 @@ static void init_julia_llvm_env(Module *m)
     jl__stack_chk_fail->setDoesNotReturn();
     add_named_global(jl__stack_chk_fail, &__stack_chk_fail);
 
-    jltrue_var = global_to_llvm("jl_true", (void*)&jl_true, m);
-    jlfalse_var = global_to_llvm("jl_false", (void*)&jl_false, m);
-    jlemptysvec_var = global_to_llvm("jl_emptysvec", (void*)&jl_emptysvec, m);
-    jlemptytuple_var = global_to_llvm("jl_emptytuple", (void*)&jl_emptytuple, m);
-    jldiverr_var = global_to_llvm("jl_diverror_exception",
-                                  (void*)&jl_diverror_exception, m);
-    jlundeferr_var = global_to_llvm("jl_undefref_exception",
-                                    (void*)&jl_undefref_exception, m);
-    jldomerr_var = global_to_llvm("jl_domain_exception",
-                                  (void*)&jl_domain_exception, m);
-    jlovferr_var = global_to_llvm("jl_overflow_exception",
-                                  (void*)&jl_overflow_exception, m);
-    jlinexacterr_var = global_to_llvm("jl_inexact_exception",
-                                      (void*)&jl_inexact_exception, m);
+    global_to_llvm("jl_true", &jl_true, m);
+    global_to_llvm("jl_false", &jl_false, m);
+    global_to_llvm("jl_emptysvec", &jl_emptysvec, m);
```

Comment:
Yes, nicely spotted. I'll rename the more specific method to `global_jlvalue_to_llvm` and cast the `&jl_emptysvec`.